docs/validator/validator-faq.md

```diff
@@ -14,32 +14,55 @@ Fisherman (or validator) - nodes that provide security by verifying the validity
 
 ## How do I become a validator?
 
-If you already have an account with sufficient amount of funds.
+You need an account with sufficient amount of funds in the wallet.
 You can follow docs [here](local-setup/running-testnet.md) how to become validators
```

Comment:
this link isn't as useful as the following 2 links

- [Becoming a Validator](/docs/validator/staking)
- [Running a Node](/docs/local-setup/running-testnet)

Reply:
note that links internal to the docs repo should start with the absolute relative path `/docs/...` to make sure that, no matter where this page gets moved to in the docs folder structure, internal links always resolve properly